WikiDBUpdaterHttp/WikiDBUpdaterHttp.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using Azure.Data.Tables;
+using WikiSentiment;
+
+
+namespace WikiDBUpdaterHttp
+{
+    /// <summary>
+    /// HTTP triggered function that updates DB with new wiki collections
+    /// </summary>
+    public class WikiDBUpdaterHttp
+    {
+        private ConfigurationWrapper config;
+
+        private string[] allLanguageCodes;
```

Comment:
It seems like this exists as a whitelist, and the only operation used is checking if it contains something specific.
In that case, a 
```suggestion
        private HashSet<string> allLanguageCodes;
```
is faster. Through the magic of Algorithms :rainbow:, you can check if a [`System.Collections.Generic.HashSet<T>`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1?view=net-7.0) contains an object in the same time, no matter if it contains zero or a billion items. A performance the string will have a hard time matching. 
For this amazing feat, you do have to accept some restrictions in how you access it, but it should not matter here.

`HashSet` also has some overhead. For very small number of items (say 4-8 probably) the raw array will beat the `HashSet`. This is usually not a concern unless you're writing very hot inner loops.